sale_order_type/views/account_move_views.xml

```diff
@@ -11,7 +11,10 @@
                     readonly="state != 'draft'"
                 />
             </label>
-            <xpath expr="//notebook//list//field[@name='sequence']" position="before">
+            <xpath
+                expr="//notebook//page[@id='invoice_tab']//list//field[@name='sequence']"
+                position="before"
+            >
```

Comment:
It can be more specific and simplified to avoid ambiguity, as follows:
```suggestion
            <xpath
                expr="//field[@name='invoice_line_ids']//list//field[@name='sequence']"
                position="before"
            >
```
I think this field might be unnecessary according to this PR: https://github.com/odoo/odoo/pull/137031. The fields used in other fields/attributes are automatically added as invisible.

@pedrobaeza @StefanRijnhart What do you think?

Reply:
Yeah, that seems so, so you can remove it directly.